src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password != null && repeatPassword != null
```

Comment:
```suggestion
        if (password != null 
```
second check is redundant